packer/ansible/roles/cyhy_dashboard/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+# tasks file for cyhy_dashboard
+
+#
+# Install curl and unzip; they are needed by cyhy_core/var/load_places.sh
+#
+- name: Install pip
```

Comment:
We already have a `pip` role at `packer/ansible/roles/pip`.  This section should be removed and `pip` should be added to the list of dependencies in `packer/ansible/roles/cyhy_dashboard/meta/main.yml`